sklearn/compose/tests/test_column_transformer.py

```diff
@@ -908,6 +911,77 @@ def test_column_transformer_remainder():
     assert ct.remainder == "drop"
 
 
+@pytest.mark.parametrize("force_int", [False, True])
+def test_column_transformer_remainder_dtypes(force_int):
+    warning_type = FutureWarning
+    warning_pattern = "force_int_remainder_cols=False"
```

Comment:
I would add a (redundant) test but specifically to show the pattern that raise the warning.